svn/common_base.py

```diff
@@ -17,12 +18,24 @@ def external_command(self, cmd, success_code=0, do_combine=False,
         env['LANG'] = svn.config.CONSOLE_ENCODING
         env.update(environment)
 
-        p = subprocess.Popen(
-            cmd,
-            stdout=subprocess.PIPE,
-            stderr=subprocess.STDOUT,
-            cwd=wd,
-            env=env)
+        if sys.platform.lower().startswith('win'):
+            startupinfo = subprocess.STARTUPINFO()
+            startupinfo.dwFlags |= subprocess.STARTF_USESHOWWINDOW
+
+            p = subprocess.Popen(
```

Comment:
Just create an empty `kwargs` variable above, add `startupinfo` to it if we're in Windows, and expand/unsplat the kwargs into the existing call rather than duplicating it.

Reply:
Done.